### DISCUSSION TITLE
Completly uninstall modoboa

### DISCUSSION THREAD

**Discussion by Colors45** [modoboa/modoboa#2777] [Help]
I rerunned the installer with the hope that this would fix some issues I had.
But this completly nuked modoboa.
How  can I completly remove modoboa and it's data so I can rerunn the installer?

**Comment by CaptainPalapa** [comment]
I need to know this, too, as I needed to correct the domain and the timezone in the installer.cfg.  Asked for domain, gave it mailx.mydomain.com and it made it mail.mailx.mydomain.com.  How do I remove modoboa so I can reinstall with the new configuration.
